src/views/count.vue

```diff
@@ -36,7 +36,7 @@
         </div>
         <ion-item>
           <ion-label position="floating">{{ $t("Stock") }}</ion-label>
-          <ion-input v-model="product.quantity"></ion-input>
+          <ion-input inputmode="numeric" v-model="product.quantity"></ion-input>
```

Comment:
I think we change type to number as well. @dt2patel WDYT?

Reply:
makes sense. then we should also set min value as well

Reply:
OK, Sir, I will fix it.